docs/actions/ssh.md

```diff
@@ -0,0 +1,264 @@
+!!! note "Target Authentication"
+
+    You can either use basic authentication or use a private key and a passphrase. 
+
+    ```json title="target example"
+    {
+        "name": "my_target",
+        "url": "tcp://my.target:4242",
+        "properties": {
+            "username": "myUsername", // (1)
+            "password": "myPassword", // (2)
+            "privateKey": "/path/to/the/private/key", // (3)
+            "privateKeyPassphrase": "myPrivateKeyPassphrase" // (4)
+        }
+    }
+    ```
+
+    1. Valid properties are `username` or `user`. Set this for basic authentication
+    2. Valid properties are `userPassword` or `password`. Set this for basic authentication
+    3. Path to private key file on the machine running Chutney
+    4. Valid properties are `privateKeyPassword` or `privateKeyPassphrase`. Set this for basic authentication
+
+# SCP
+
+## Download
+
+=== "Inputs"
+
+    | Required | Name          | Type                                               | Default |
+    |:--------:|:--------------|:---------------------------------------------------|:-------:|
+    |    *     | `target`      | String                                             |         |
+    |    *     | `source`      | String                                             |         |
+    |    *     | `destination` | String                                             |         |
+    |          | `timeout`     | String ([Duration](/actions/other/#duration-type)) | `5 sec` |
+
+SCP download has no outputs
+
+### Example
+
+=== "Kotlin"
+``` kotlin
+ScpDownloadTask(
+    target = "myTarget",
+    source = "/path/to/the/source/file", // (1)
+    destination = "/path/to/the/destination/file", // (2)
+    timeout = "42 ms"
+)
+```
+
+1. The file to download from the target machine
+2. The path on the machine running Chutney
```

Comment:
1. File relative path on the target machine
2. Path where to save file on local machine
?

Reply:
It should be absolute an path since there is no way to know from where to start.
We could add a default folder as a feature, but for now there is none.